community-operators/cluster-logging/clusterlogging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,416 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: clusterlogging.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "openshift optional, logging, aggregated, elasticsearch, kibana, fluentd, efk"
+    certified: false
```

Comment:
This needs to be a string, ie: `"false"`

Reply:
updated